include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp

```diff
@@ -160,6 +160,23 @@ class Orbit : public Trajectory
         const Instant& anEpoch, const Length& anAltitude, const Shared<const Celestial>& aCelestialObjectSPtr
     );
 
+    /// @brief              Constructs a geosynchronous orbit
+    ///
+    ///                     Model: Kepler (J2 Perturbation).
+    ///
+    /// @param              [in] anEpoch An orbit epoch
+    /// @param              [in] anInclination An orbit inclination
+    /// @param              [in] aLongitude A longitude above the surface
+    /// @param              [in] aCelestialObjectSPtr A shared pointer to a central celestial body
+    /// @return             Circular orbit
+
+    static Orbit GeoSynchronous(
```

Comment:
In `Geosynchronous` (see [wikipedia](https://en.wikipedia.org/wiki/Geosynchronous_orbit)) the `S` is not capitalized. Could you adjust the codebase please?

Reply:
@lucas-bremond What about aligning with SunSynchronous?

Reply:
Yes fair, dropping the case ;)